### ISSUE TITLE
[Question] Use Encoder from Native Android

### ISSUE THREAD

**Issue by tsoen** [BinomialLLC/basis_universal#308]
Hello,

This is probably a stupid question, but is it possible to use / compile file sources to **encode** files to .basis files directly from Android ?
For example building a shared library .so, that can be called from Android.
(Use case is to be able to pick any image on the fly on Android and to encode it to .basis format)

Sorry if I'm totally off the point here.
Have a good day

**Comment by richgel999** [comment]
Yes, it should be, however we have not tried this yet. The code is cross platform enough to compile on Linux/OSX/Windows/Emscripten - so it should work.

**Comment by tsoen** [comment]
Hello,

Thank you for your reply.
Any advice or direction on how to do this ?
I have downloaded the source code but have not been able to configure the project to change to output type (shared library in my case).

Regards
